src/boot/efi/meson.build

```diff
@@ -62,6 +62,8 @@ if have_gnu_efi
         efi_conf = configuration_data()
         efi_conf.set_quoted('PACKAGE_VERSION', meson.project_version())
         efi_conf.set_quoted('EFI_MACHINE_TYPE_NAME', EFI_MACHINE_TYPE_NAME)
+        efi_conf.set('SD_BOOT_LOG_TPM', get_option('tpm'))
```

Comment:
Ah, OK. Somehow I thought that works differently. That means that quite a few unnecessary `if` can be removed.